src/main/java/seedu/address/model/person/patient/Height.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.model.person.patient;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Patient's height in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidHeight(String)}
+ */
+public class Height {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Height should be a positive decimal point number representing the patient's height in metres "
```

Comment:
By the way, should we account for the diagnosis beginning with white spaces too? Or will our logic trim away leading and trailing whitespaces?

Reply:
Duplicate comment